app/src/main/java/org/gem/indo/dooit/views/main/fragments/target/adapters/TargetPagerAdapter.java

```diff
@@ -57,4 +58,10 @@ public void updateGoals(List<Goal> goals) {
         this.goals = goals;
         notifyDataSetChanged();
     }
+
+    @Override
+    public int getItemPosition(Object object) {
+        return POSITION_NONE;
```

Comment:
@FredJEhlers Could you please write a comment here explaining why we had to do this, and what the implications are? It doesn't have to be terribly long. Just mention that the `PagerAdapter` does not re-instantiate the current item, and that `POSITION_NONE` forces the `ViewPager` to recreate all the pages.